release-notes-5.7.0.md

```diff
@@ -4,5 +4,6 @@
 ${version-number}
 
 #### New Features
+- US590035: Added a second image which uses Java Util Logging
```

Comment:
Maybe update this to say two images generated one image configured with CAF Logging, the other image falls back to Tomcat JULI logging.